app/src/components/tags/VerticalTagViewer.vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
+import { DocType, db, type TagDto, type queryOptions as options } from "luminary-shared";
+import { ref, watch } from "vue";
+import { useRouter } from "vue-router";
+
+const router = useRouter();
+type Props = {
+    tag?: TagDto;
+    title?: string;
+    queryOptions: options;
+    showPublishDate?: boolean;
+};
+const props = withDefaults(defineProps<Props>(), {
+    showPublishDate: true,
+});
+
+const isContentSelected = (slug: string) => {
+    if (router.currentRoute.value.params.slug === slug) return true;
+    return false;
+};
+
+const taggedDocs = db.contentWhereTagAsRef(props.tag?._id, props.queryOptions);
+const tagContent = props.tag
+    ? db.whereParentAsRef(props.tag._id, DocType.Tag, props.queryOptions.languageId, [])
+    : ref([]);
+
+const tagTitle = ref(props.title);
+const tagSummary = ref("");
+
+watch(tagContent, () => {
+    if (props.title) {
+        tagTitle.value = props.title;
+        return;
+    }
+
+    if (tagContent.value.length > 0) {
+        tagTitle.value = tagContent.value[0].title;
+        tagSummary.value = tagContent.value[0].summary || "";
+        return;
+    }
+
+    tagTitle.value = "No translation found";
+});
+</script>
+
+<template>
+    <div>
+        <div>
+            <RouterLink
+                v-for="content in taggedDocs"
+                :key="content._id"
+                :to="{
+                    name: 'content',
+                    params: { slug: content.slug },
+                }"
+            >
+                <div
+                    class="flex justify-between border-l-4 border-transparent bg-opacity-20 px-2 py-[2px] transition duration-200 hover:border-opacity-100 hover:bg-yellow-300 hover:bg-opacity-10"
```

Comment:
Same point on using custom values (`py-[2px]`), there should be a good reason for it. In this case, `0.5` can be used instead of `2px`, as they are essentially the same value.

```suggestion
                    class="flex justify-between border-l-4 border-transparent bg-opacity-20 px-2 py-0.5 transition duration-200 hover:border-opacity-100 hover:bg-yellow-300 hover:bg-opacity-10"
```